packages/omgidl-parser/src/IdlNodes/IdlNode.ts

```diff
@@ -0,0 +1,191 @@
+import { ConstantValue } from "@foxglove/message-definition";
+
+import { BaseASTNode, ConstantASTNode, EnumASTNode } from "../astTypes";
+import { SIMPLE_TYPES, normalizeType } from "../primitiveTypes";
+import { IDLMessageDefinition, IDLMessageDefinitionField } from "../types";
+
+export class IdlNode<T extends BaseASTNode = BaseASTNode> implements BaseASTNode {
```

Comment:
`Ast`?

I mean I think it looks terrible, but at least it would be consistent with `Idl` 😄

Reply:
all caps reigns supreme now